src/cephadm/cephadm.py

```diff
@@ -7109,6 +7109,13 @@ def command_rm_daemon(ctx):
          verbosity=CallVerbosity.DEBUG)
     call(ctx, ['systemctl', 'disable', unit_name],
          verbosity=CallVerbosity.DEBUG)
+
+    # force remove rgw admin socket file if leftover
+    if daemon_type in ['rgw']:
+        rgw_asok_path = f'/var/run/ceph/{ctx.fsid}/ceph-client.{ctx.name}.*.asok'
+        call(ctx, ['rm', '-rf', rgw_asok_path], 
```

Comment:
```
cephadm.py:7116:48: W291 trailing whitespace
1     W291 trailing whitespace
```
flake8 issue, just need to remove the whitespace at the end of the line

Reply:
updated!